packages/cactus-plugin-satp-hermes/package.json

```diff
@@ -78,7 +78,16 @@
     "watch": "tsc --build --watch",
     "forge": "forge build ./src/solidity/*.sol --out ./src/solidity/generated",
     "forge:test": "forge build ./src/test/solidity/contracts/*.sol --out ./src/test/solidity/generated",
-    "forge:all": "run-s 'forge' 'forge:test'"
+    "forge:all": "run-s 'forge' 'forge:test'",
+    "db:setup": "bash -c 'npm run db:destroy || true && run-s db:start db:migrate db:seed'",
```

Comment:
I did many contributions to this code. Unfortunately the squash you did removed that history.

Please add me as a co-author to the squashed commit: 
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reply:
added you as a co-author, I didn't know about it and squash removed the co-author my bad!